bin/dart_openapi_codegen.dart

```diff
@@ -423,6 +425,10 @@ class SingletonSchema extends Schema {
     'string/uri': SingletonSchema('Uri',
         fromJson: (x) => 'Uri.parse($x as String)',
         toJson: (x) => '$x.toString()'),
+    // nico wanted this repetition
+    'string/mx-mxc-uri': SingletonSchema('Uri',
+        fromJson: (x) => 'Uri.parse($x as String)',
+        toJson: (x) => '$x.toString()'),
```

Comment:
Can we add extra validation, that it actually has a scheme of "mxc://"?

Reply:
done